### DISCUSSION TITLE
I am a verified teacher but have not been able to access the copilot. What should I do?

### DISCUSSION THREAD

**Discussion by SebaGorga** [community/community#48369] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/125015786/220877031-a0438fc3-f154-4cd4-a514-8967590cb700.png)

This is what I see in the copilot page.

**Comment by Akash1134** [comment, marked as answer]
Hi @SebaGorga ,

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.

Please reverify yourself as a teacher, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

**Reply by SebaGorga** [reply]
I have tried to resubmit but it does not let me click on "Continue". It is telling me that my email "has already been used to obtain an academic discount on GitHub account(s)" and that "You were last verified as a teacher on Feb 23, 2023. It is not necessary for you to re verify at this time."

**Reply by Akash1134** [reply]
Thanks for the quick update. In that case, you will need to cancel your trial subscription, and then go to https://github.com/github-copilot/free_signup to claim your student subscription.

**Reply by SebaGorga** [reply]
It worked. Thank you!

**Comment by ozak** [comment]
Hi,

I am facing the same issue. It said I was supposed to get free access, but now I am stuck without being able to start a free trial or to cancel it. what can I do?

<img width="848" alt="image" src="https://user-images.githubusercontent.com/650534/229892686-19c61a93-468d-49df-b633-0d25d462bc2b.png">

**Comment by fdbf** [comment]
@Akash1134 

Same as @ozak here. Just got verified as teacher. I canceled the free trial. It still tells me that I have 52 days left. 

![image](https://user-images.githubusercontent.com/19819011/230876883-637f5918-33e5-4f8f-ba31-ecb60cbc4fa6.png)

However, when I click on the provided link [https://github.com/github-copilot/free_signup](https://github.com/github-copilot/free_signup) I get this screen.

![github](https://user-images.githubusercontent.com/19819011/230876682-3d326d4c-b66a-41a6-9302-5bf0b82a641a.jpg)

Any help is much appreciated.

**Comment by Akash1134** [comment]
Hi @fdbf and @ozak , sorry for the trouble caused. Our Copilot team is currently looking into this issue

**Reply by genglyu** [reply]
Fingers crossed for a solution.
Although the teacher verification does allow me to submit my information again.

Did the copilot team figure out a solution?

Asking 5 months later.

**Comment by LuisJoseSanchez** [comment]
Hello all,

Same problem here.

I'm a verified teacher but when I go to Settings -> Copilot, I can't enable Copilot. I got the screen offering to start a free trial.

![image](https://github.com/community/community/assets/840797/af97d68e-5d8a-470d-8889-5952ec6edfdf)

**Comment by genglyu** [comment]
Yep, same problem here.
Fingers crossed for a solution.
Although the teacher verification does allow me to submit my information again.

**Comment by kolgenpav** [comment]
Is Copilot free for verified teachers? Now it seems not :(
![image](https://github.com/community/community/assets/3978552/629fc11b-7e72-45ce-af88-0aef1a5e4a8e)

**Reply by davecheney** [reply]
@kolgenpav i've checked and it doesn't look like github knows you're an educator. you can apply here https://education.github.com/discount_requests/application

**Reply by kolgenpav** [reply]
Thank You,
My request to [](https://education.github.com/discount_requests/application) was approved, 
![image](https://github.com/community/community/assets/3978552/fe0d7262-43f4-4671-a2ff-d62f9b985fdc)

but when I try to use GitHub Copilot in IntelliJ IDEA, I see
![image](https://github.com/community/community/assets/3978552/1ebb274e-a244-4888-80de-2813f21ee51d)

**Comment by ChenChenGith** [comment]
**I have the same problem。**

**I start the trail of the free 30 days of copilot and then cancel it. Now I have 30 days, and in the Setting-Copilot pages, It seems that every thing is ok. But I do not know if I can continue using copilot in 30 days. I have be approved as a teacher:**

![image](https://github.com/community/community/assets/45627234/b14343f3-f4ae-47df-b9a5-811a2109017a)
![image](https://github.com/community/community/assets/45627234/05b86228-f95c-4eae-a1e4-aeac7fb50550)
![image](https://github.com/community/community/assets/45627234/490cd8f3-7948-4804-82cf-a54fbf9e3381)

**Comment by kolgenpav** [comment]
Hi,
I want to inform that after some period I have use GitHub Copilot in my IDE. It may be important, I see difference in color of my approved label at https://education.github.com/discount_requests/application

![image](https://github.com/community/community/assets/3978552/76232de4-7c12-4f96-8155-b28a58eb18a2)

I want to thank GitHub command for the opportunity to use this tool. It;'s a great tool and I think we as teachers must suggest to use it our students.

**Reply by ChenChenGith** [reply]
Thank you! It seems that I need to wait for some time to get the benefits.

**Comment by mbehrisch** [comment]
Hi, I am verified as a teacher. 
![Screenshot - 16-2-2024 , 14_43_14](https://github.com/community/community/assets/310635/8d4eb8bd-31f7-4ade-9413-26ce937fab39)

But my Github Copilot page (https://github.com/settings/copilot) does not allow me to enable Copilot. 
The only options are 1) "Buy Copilot"
![Screenshot - 16-2-2024 , 14_46_59](https://github.com/community/community/assets/310635/57d3c26a-039c-42c0-bc83-f75a3f5c5057)

2) and get from an organization, but my University is not showing up in this list only in the "Enterprises". However, option 2 points me to the education program. :)

NB: https://github.com/github-copilot/signup states that: 
![Screenshot - 16-2-2024 , 14_51_02](https://github.com/community/community/assets/310635/e7236a4c-125a-4543-bcd2-5c46f900dc24)
Is this a related or unrelated issue?

Any idea how this can be fixed?
Michael Behrisch

**Reply by mbehrisch** [reply]
@Akash1134 could you help with this, please?

**Reply by mbehrisch** [reply]
**Closing this question here**, as it is a dangling (and answered) thread. Reopening under: https://github.com/orgs/community/discussions/109032

**Comment by kolgenpav** [comment]
I was able to use Copilot in the IDE after some time after verification
(2-3 days).

[image: image.png]

Sincerely,
Hennadii

нд, 18 лют. 2024 р. о 23:05 Michael Behrisch ***@***.***>
пише:

**Comment by trlinden** [comment]
Same problem here. It says that my application was approved as a teacher on the right hand side of this page: https://education.github.com/discount_requests/application

But when I go to https://github.com/github-copilot/signup, it only gives me the option to pay $10/month for an individual subscription. It has been approved for at least 24 hours now, so I would have guessed it would have percolated through the system.

**Reply by xdefago** [reply]
I'm in the exact same situation.
Apparently, according to some of the comments above, the approved icon changes color (from green to purple) a few days after the approval and that's when the benefits (e.g., copilot) become available.

I don't know if that's true, because mine was approved three days ago and didn't change color yet. Also, I got no feedback that my application was uploaded on the website but my application was magically there when I looked in the settings.
Maybe it's only me, but I don't think that github could make a verification process that is more confusing, less informative, and overall more painful even if they tried very very hard...

**Comment by mbehrisch** [comment]
For me the situation (magically) improved without my further involvement or information flow.

Greetings,
Michael

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by myuan27** [comment]
Same problem here, has verified as teacher but still shows "You don't have access to GitHub Copilot. Sign up by visiting https://github.com/settings/copilot."
![teacher_github](https://github.com/community/community/assets/156099600/d1117a2a-554e-4546-b932-9b397d04ba2b)

**Reply by w-poh** [reply]
Did you find a solution or is anybody aware how to solve this? 
I have the exact same problem: Applied as a teacher, badge with "Approved" changed to purple, it says now I have full benefits but in the github.com settings, it proposes me to do the free trial but not my organization. My colleagues did it the same way but the have all copilot stuff on this side [https://github.com/settings/copilot]. Looks like this: 
![image](https://github.com/user-attachments/assets/65b4fc6f-c61a-41da-b8ae-c6859cc24490)

**Reply by trlinden** [reply]
it ended up working automatically for me after about 24 hours -- not sure why there is a delay, since it isn't mentioned.

**Reply by abarthele** [reply]
I have the same problem, and still no idea how to make it work, or even if it should work. 

The GitHub Education enrollment page states that I have been verified as "faculty" (is that different from "teacher"?), and I have a purple badge "Approved", but I can't activate Copilot...

**Reply by w-poh** [reply]
@abarthele For me, it was my fault: In my settings somewhere (I guess in Copilot section) I had to agree to some terms and conditions or so. After that it worked immediately.

**Reply by abarthele** [reply]
Thanks @w-poh ! It turns out that in my settings > Billing and plans > Plans and usage, I needed to explicitly change the current plan from the free version to Github Pro (also free in my case). This enabled Copilot Pro.
